static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -106,6 +106,12 @@ define([
         }
     }
 
+    function setFluid(el) {
+        if (el.classList.contains('ad-slot--container-inline')) {
+            el.classList.add('ad-slot__fluid250');
```

Comment:
Have you tried with `ad-slot--fluid`? the `ad-slot__fluid250` has lots of side-effects which may be problematic in the long term.